### ISSUE TITLE
SVG animations

### ISSUE THREAD

**Issue by jonathan-reisdorf** [gagarine/no-transition#20]
SVG smil animations are still playing. Firefox has an `image.animation_mode` flag in the `about:config` that can be set from `normal` to `none` - maybe your addon could also make use on that? (might also affect gifs though, but maybe for some that's also desirable)
On the other hand, it's probably not possible to pause/continue these animations without page reload, and afaik with the flag we don't get the final state of the animations but they don't even start, so a custom solution might be required. Hm, hope it doesn't turn out to be too much work.
In general, your addon does a very good job disabling the animations already :)

**Comment by ghost** [comment]
https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/browserSettings/imageAnimationBehavior

**Comment by jonathan-reisdorf** [comment]
Thanks, so the `once` setting is much better for our purpose.

**Comment by jonathan-reisdorf** [comment]
Yes but that still only gives us the choice between fully playing and stopping animations, fully playing and repeating them or not even starting at all. Perfect would be a way to skip directly to the last "frame" of the animation
